scripts/votemanager/VoteManagerControls.tsx

```diff
@@ -65,13 +65,13 @@ export default class VoteManagerControls extends Component<any, any> {
   }
 
   setSelectedToContrib() {
-    const payoutcontrib = _.uniq([...groups.gdt.data, ...groups.elite.data, ...groups.shw.data, ...groups.builders.data, 'stellardynamic']);
+    const payoutcontrib = _.uniq([...groups.ascend.data, ...groups.builders.data, ...groups.dutchpool.data, ...groups.gdt.data, ...groups.lig.data, ...groups.shw.data, "acheng", "adrianhunter", "badman0316", "bluedragon", "carolina", "chamberlain", "communitypool", "crodam", "devasive", "eastwind_ja", "elevate", "elum", "endro", "forger_of_lisk", "fulig", "grajsondelegate", "gregorst", "honeybee", "iii.element.iii", "leo", "liskjp", "liskpoland.pl", "liskpool.top", "liskpool_com_01", "luxiang7890", "minions", "moracle.network", "mrgr", "panzer", "phinx", "prolina", "samuray", "savetheworld", "seven", "shinekami", "someonesomeone", "spacetrucker", "stellardynamic", "tonyt908", "ultrafresh", "vipertkd", "vrlc92", "will", "xujian"]);
```

Comment:
Too opinionated, please revert this part and start another pull request for this.

Reply:
Hey @5an1ty - I'm fine with reverting it, but this breaks the vote manager wizard as it does not fit into 101. Open for suggestions here, didn't want to remove stellardynamic, but also think the previous list does not really represent contributors.

Happy to discuss this in a different PR though.

Reply:
I agree on reverting the data. The proposed changes are really opinionated, which in my opinion are very arguable.

Anyway, the builders data should be got smaller by now.